cornflow-dags/DAG/two_dimension_bin_packing/schemas/instance_checks.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "properties": {
+    "non_valid_objects": {
+      "type": "array",
+      "title": "Items that are bigger than the container",
+      "is_warning": true,
+      "items": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "integer"
+          }
+        },
+        "required": [
+          "id"
+        ],
+        "additionalProperties": false
+      }
+    },
+    "non_valid_data": {
+      "type": "object",
+      "properties": {
+        "valid": {
+          "type": "boolean"
+        }
+      }
+    }
```

Comment:
It's actually needed if you plan on developing an interface on cornflow-app, because the app uses the `required` list to order the columns of the table, and I'm pretty sure it will raise an error if this list doesn't exist. (Since the last changes I made on the app, the instance and solution checks are shown as tables, just as the instance and solution. The tables use the same classes than the instance and solution tables, so they also need the required lists.)
If you don't plan on developing an interface on cornflow-app, you're right, it isn't needed. 
If you think we eventually might make one, you can always define it as an empty list. 

Or maybe we could update cornflow-app so that it doesn't need the `required` list for the tables, and assumes that they are empty if they are not provided. I think it would make sense.

Reply:
I think in the future it would make sense to change this feature of the app so that it doesn't consider that all properties of a config table are required. It might cause problems in cases like this